src/CardBack.js

```diff
@@ -16,35 +16,35 @@ class CardBack extends Component {
         const height = this.props.chosenPlayer.Height
         const feet = Math.floor(height / 12)
         const inches = (height - (feet * 12))
-        const heightWeight = `${feet}' ${inches}", ${this.props.chosenPlayer.Weight}lbs`
+        const heightWeight = `He is ${feet}' ${inches}" tall and weighs ${this.props.chosenPlayer.Weight}lbs`
         return heightWeight
```

Comment:
Some day maybe I'll have to change these pronouns to a variable. Wouldn't that be a cool problem to have?